### PR TITLE
Checker checking also DB and S3

### PR DESCRIPTION
**Pull request by procrastinatio**
Check carefully, since this route is used by varnish to keep backend online or not.

See https://github.com/geoadmin/mf-chsdi3/issues/3028

### CODE REVIEW

File: chsdi/views/checker.py [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+class Checker(FileView):
 
     def __init__(self, request):
         self.request = request
+        self.dynamodb_table_name = 'geoadmin-file-'
+        self.bucket_key_name = 'geoadmin_file_storage_bucket66'
+        self.bucket_name = request.registry.settings['geoadmin_file_storage_bucket']
+        FileView.__init__(self, request)
 
     @view_config(route_name='checker')
```
Comment (danduk82) [inline chsdi/views/checker.py]: similarly to what we have done for `service-proxywms`, `/checker` should only check the sanity of mf-chsdi3, without the backends, and there should be a new route `/backend_checker` that checks the backends (and implicitely the sanity of `mf-chsdi3`.

**Comment by danduk82** [comment]
@procrastinatio , is it realistic to keep this PR in the milestone that has to be merged before tomorrow?

File: chsdi/views/checker.py [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+from pyramid.httpexceptions import HTTPInternalServerError
+from chsdi.views.files import FileView
 
 
-class Checker(object):
+class Checker(FileView):
 
     def __init__(self, request):
         self.request = request
+        self.dynamodb_table_name = 'geoadmin-file-'
```
Comment (danduk82) [inline chsdi/views/checker.py]: @procrastinatio , I am not sure about the name of the table here, shouldn't it be `geoadmin-file-storage`? Can you verify please?

Comment (procrastinatio) [reply chsdi/views/checker.py]: Actually it is not used

File: chsdi/views/checker.py (hunk 1 above)
Comment (procrastinatio) [reply chsdi/views/checker.py]: OK, done

File: chsdi/views/checker.py [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
+        self.bucket_key_name = 'geoadmin_file_storage_bucket66'
+        self.bucket_name = request.registry.settings['geoadmin_file_storage_bucket']
+        FileView.__init__(self, request)
 
     @view_config(route_name='checker')
     def home(self):
+        try:
+            self.dynamodb_fileshandler.table.count()
```
[the remark this answers is not in the document]
Comment (procrastinatio) [reply chsdi/views/checker.py]: No, S3 check is unten

[the remark this answers is not in the document]
Comment (danduk82) [reply chsdi/views/checker.py]: ah ok sorry

**Pull request merged**